### ISSUE TITLE
Getting error when running "gem install curb -v '0.9.3'"

### ISSUE THREAD

**Issue by BoazWarshawsky** [bundler/bundler#6995]
I am running a pipeline using Azure DevOpes pipeline on host Mac machine OS X 10.13.6 (17G5019) High Sierra.

My environment using:
Node.js: v8.12.0
ruby: 2.3.5

The issue:
When I am running this ruby command `gem install curb -v '0.9.3'` I am getting error.

Error log:
creating curb_config.h
creating Makefile

current directory: /Users/vsts/.rvm/gems/ruby-2.3.5/gems/curb-0.9.3/ext
make "DESTDIR=" clean

current directory: /Users/vsts/.rvm/gems/ruby-2.3.5/gems/curb-0.9.3/ext
make "DESTDIR="
compiling curb.c
compiling curb_easy.c
curb_easy.c:961:3: warning: implicit conversion loses integer precision: 'long' to 'int' [-Wshorten-64-to-32]
CURB_IMMED_SETTER(ruby_curl_easy, proxy_type, -1);
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./curb_macros.h:110:27: note: expanded from macro 'CURB_IMMED_SETTER'
ptr->attr = NUM2LONG(attr); 
~ ^~~~~~~~~~~~~~
/Users/vsts/.rvm/rubies/ruby-2.3.5/include/ruby-2.3.0/ruby/ruby.h:665:21: note: expanded from macro 'NUM2LONG'
#define NUM2LONG(x) RB_NUM2LONG(x)
^~~~~~~~~~~~~~
/Users/vsts/.rvm/rubies/ruby-2.3.5/include/ruby-2.3.0/ruby/ruby.h:664:24: note: expanded from macro 'RB_NUM2LONG'
#define RB_NUM2LONG(x) rb_num2long_inline(x)
^~~~~~~~~~~~~~~~~~~~~
curb_easy.c:3207:34: warning: implicit conversion loses integer precision: 'long' to 'CURLoption' [-Wshorten-64-to-32]
curl_easy_setopt(rbce->curl, option, NUM2LONG(value));
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/Cellar/curl/7.63.0/include/curl/curl.h:2813:68: note: expanded from macro 'curl_easy_setopt'
#define curl_easy_setopt(handle,opt,param) curl_easy_setopt(handle,opt,param)
~~~~~~~~~~~~~~~~ ^~~
2 warnings generated.
compiling curb_errors.c
curb_errors.c:324:10: error: duplicate case value 'CURLE_PEER_FAILED_VERIFICATION'
case CURLE_SSL_CACERT: /* 60 - problem with the CA cert (path?) _/ ^ /usr/local/Cellar/curl/7.63.0/include/curl/curl.h:603:26: note: expanded from macro 'CURLE_SSL_CACERT' #define CURLE_SSL_CACERT CURLE_PEER_FAILED_VERIFICATION ^ curb_errors.c:297:10: note: previous case defined here case CURLE_SSL_PEER_CERTIFICATE: /_ 51 - peer's certificate wasn't ok */
^
/usr/local/Cellar/curl/7.63.0/include/curl/curl.h:610:36: note: expanded from macro 'CURLE_SSL_PEER_CERTIFICATE'
#define CURLE_SSL_PEER_CERTIFICATE CURLE_PEER_FAILED_VERIFICATION
^
1 error generated.
make: *** [curb_errors.o] Error 1

make failed, exit code 2

Gem files will remain installed in /Users/vsts/.rvm/gems/ruby-2.3.5/gems/curb-0.9.3 for inspection.
Results logged to /Users/vsts/.rvm/gems/ruby-2.3.5/extensions/x86_64-darwin-17/2.3.0/curb-0.9.3/gem_make.out
##[debug]Exit code 1 received from tool '/bin/bash'
##[debug]STDIO streams have closed for tool '/bin/bash'
##[error]Bash exited with code '1'.
##[debug]Processed: ##vso[task.issue type=error;]Bash exited with code '1'.
##[debug]task result: Failed
##[debug]Processed: ##vso[task.complete result=Failed;done=true;]
##[section]Finishing: Prepare android workspace using app version 1c9aae87-364f-48f3-9dee-531c1ba13b2c

**Comment by colby-swandale** [comment]
This issue does not relate to Bundler. I suggest checking your error log and look for why the library is not being compiled correctly.

```
curb_errors.c:324:10: error: duplicate case value 'CURLE_PEER_FAILED_VERIFICATION'
```

**Comment by BoazWarshawsky** [comment]
Problem solved when using curb version 0.9.7
